src/platform/common/process/proc.node.ts

```diff
@@ -25,9 +24,8 @@ import {
 } from './types.node';
 
 export class BufferDecoder implements IBufferDecoder {
-    public decode(buffers: Buffer[], encoding: string = DEFAULT_ENCODING): string {
-        encoding = iconv.encodingExists(encoding) ? encoding : DEFAULT_ENCODING;
-        return iconv.decode(Buffer.concat(buffers), encoding);
+    public decode(buffers: Buffer[]): string {
+        return Buffer.concat(buffers).toString(DEFAULT_ENCODING);
```

Comment:
The enconding was always set to `utf-8`, which meant iconv-lite was never necessary